app/statistics/[cycle]/page.tsx

```diff
@@ -0,0 +1,52 @@
+import AdmissionsCycleStatistics from '@/components/AdmissionsCycleStatistics'
+import prisma from '@/db'
+import { Decision, NextAction } from '@prisma/client'
+
+export const dynamic = 'force-dynamic'
+
+export default async function AdmissionsCycleStatisticsPage({
+  params
+}: {
+  params: { cycle: string }
+}) {
+  const cycle = parseInt(params.cycle)
+
+  const outcomes = await prisma.outcome.findMany({
+    where: {
+      application: {
+        admissionsCycle: cycle
+      }
+    }
+  })
+
+  const noApplications = outcomes.length
+  const noOffers = outcomes.filter((outcome) => outcome.decision === Decision.OFFER).length
+  const noRejections = outcomes.filter((outcome) => outcome.decision === Decision.REJECT).length
```

Comment:
(Minor) I think we can afford full readability in most cases. `ApplicationsCount`, `offersCount`, `rejectionsCount` read less ambiguously to me, while it took me the parsing of the whole `filter` to understand that `noOffers` mean number-of offers and not literally "no offers"